src/gr_rtl_tuner.cpp

```diff
@@ -387,6 +387,18 @@ void rtl_start_fm(rtl_ctx_t* tuner)
         return;
     }
 
+}
+
+// Wait for the gnuradio threads to terminate
+// Part of the external API
+// @param tuner The tuner context
+void rtl_wait(rtl_ctx_t* tuner)
+{
+    if (tuner == NULL)
+    {
+        return;
+    }
+
     tuner->top_block->start();
```

Comment:
I'm confused about why top_block->start is in the wait function now, shouldn't that still be in rtl_start_fm?

Reply:
Because I wasn't thinking clearly.  Should be fixed now.